src/sphinx_notion/writers.py

```diff
@@ -77,7 +77,7 @@ def convert_inline_elements(node: nodes.Node) -> dict[str, Any]:
         # node is Text
         return {
             "type": "text",
-            "text": {"content": node.astext().strip(" ")},
```

Comment:
I don't understand why this `strip` was here.

My test document looks good without it.